zkevm-circuits/src/tx_circuit.rs

```diff
@@ -79,11 +80,11 @@ use halo2_proofs::plonk::SecondPhase;
 use itertools::Itertools;
 
 /// Number of rows of one tx occupies in the fixed part of tx table
-pub const TX_LEN: usize = 24;
+pub const TX_LEN: usize = 27;
```

Comment:
This will also change if you remove that field `AccessListGasCost`

Reply:
Decrease 1 (to `26`) in commit [6880378](https://github.com/scroll-tech/zkevm-circuits/pull/1045/commits/68803783b1447b8fefe2c1fabd626beae51b8a83).